usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -1781,6 +1784,60 @@ static CK_RV make_wrapblob(STDLL_TokData_t * tokdata, CK_ATTRIBUTE * tmpl_in,
     return rc;
 }
 
+static void *ep11_load_host_lib()
+{
+    void *lib_ep11;
+    char *ep11_lib_name;
+    char *errstr;
+
+    ep11_lib_name = getenv(EP11SHAREDLIB_NAME);
+    if (ep11_lib_name != NULL) {
+        lib_ep11 = dlopen(ep11_lib_name, RTLD_GLOBAL | RTLD_NOW);
+
+        if (lib_ep11 == NULL) {
+            errstr = dlerror();
+            OCK_SYSLOG(LOG_ERR,
+                       "%s: Error loading shared library '%s' [%s]\n",
```

Comment:
warning instead of error ? Or better just a debug trace and if all attempts to load an ep11 lib fail one error syslog entry? You know what I mean: an admin just seeing the ERROR with no knowledge about the way how the ep11 token comes up will be concerned without a reason.

Reply:
oh sorry -i've been too quick. This first load is for a set environment variable, and should of course be an error if faileed and you did already the other logs as debug.